### ISSUE TITLE
Add Non Linear Activation functions to PyTorch Frontend nn.functional

### ISSUE THREAD

**Issue by ankitdsi2010** [unifyai/ivy#4348]
Add [non linear activation functions](https://pytorch.org/docs/stable/nn.functional.html#non-linear-activation-functions) to PyTorch frontend nn.functional:

- [ ] threshold
- [ ] relu
- [ ] hardtanh
- [ ] hardswish
- [ ] relu6 
- [ ] elu
- [ ] selu
- [ ] celu
- [ ] leaky_relu
- [ ] prelu
- [ ] rrelu
- [ ] glu
- [ ] gelu
- [ ] logsigmoid
- [ ] hardshrink
- [ ] tanhshrink
- [ ] softsign
- [ ] softplus
- [ ] softmin
- [ ] softmax
- [ ] softshrink
- [ ] gumbel_softmax
- [ ] log_softmax
- [ ] tanh
- [ ] sigmoid
- [ ] hardsigmoid
- [ ] silu
- [ ] mish
- [ ] batch_norm
- [ ] group_norm
- [ ] instance_norm
- [ ] layer_norm
- [ ] local_response_norm
- [ ] normalize
